manifest/yaml.go

```diff
@@ -113,7 +113,29 @@ func (v *Resource) SetName(name string) error {
 }
 
 func (v Services) MarshalYAML() (interface{}, error) {
-	return nil, fmt.Errorf("unimplemented")
+	services := make(map[string]interface{})
+
+	// Loop over all of the services
+	for _, s := range v {
+		service := make(map[string]interface{})
+
+		// If there is a build key
```

Comment:
I think it's OK to always write out the build key rather than trying to do this sort of detection.

A YAML marshal of a Praxis manifest.Manifest can be an incredibly verbose yaml laden with all of the defaults, etc fully hydrated.